MDLIB/InputOutput.c

```diff
@@ -1087,7 +1087,7 @@ Input ReadInputFile( char *fileName ) {
     model.penalty            = ReadDou2( fin, "penalty",           1.0e3 ); // Penalty factor
     model.auto_penalty       = ReadDou2( fin, "auto_penalty",        0.0 ); // Activates automatic penalty factor computation
     model.diag_scaling       = ReadInt2( fin, "diag_scaling",          1 ); // Activates diagonal scaling
-    model.preconditioner     = ReadInt2( fin, "preconditioner",        0 ); // Preconditoner type for Newton iterations, 0: Picard preconditionner
+    model.preconditioner     = ReadInt2( fin, "preconditioner",        0 ); // Preconditoner type for Linear solver, 0: Picard preconditionner, -1: symmetrised Picard, 2: symmetrised Newton
```

Comment:
In the code I see only options -1, 0, 1, I don't see option 2.

Reply:
correct, fixed